AgentInterfaces/MaintenanceSchedule.cs

```diff
@@ -43,13 +48,14 @@ public class MaintenanceEvent
 
         public string ResourceType { get; set; }
 
-        [JsonProperty("Resources")]
-        public IList<string> AffectedResources { get; set; }
```

Comment:
we should not change property names, these classes can (and are) used in a number of places.

Reply:
Didn't realize this project was pulled into the production agent. I'll revert this and move other changes out of here